fileio/ft_read_data.m

```diff
@@ -1372,6 +1372,21 @@
       dat = read_yokogawa_data(filename, hdr, begsample, endsample, chanindx);
     end
     
+  case 'blackrock_nsx'
+    % use the NPMK toolbox for the file reading
+    ft_hastoolbox('NPMK', 1);
+    
+    % ensure that the filename contains a full path specification,
+    % otherwise the low-level function fails
+    [p,f,e] = fileparts(filename);
+    if ~isempty(p)
+      % this is OK
+    elseif isempty(p)
+      filename = which(filename);
+    end
+    orig = openNSx(filename, 'duration', [begsample endsample], 'channels', chanindx);
+    dat  = double(orig.Data);
```

Comment:
I thought that I had access to a paused file, but it turns out I don't. It would be great if the support staff could chime in here.